From 41e00cf5a1fe47153c043a947a225b8eacd3fb04 Mon Sep 17 00:00:00 2001
From: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Date: Fri, 8 Mar 2019 15:26:47 +0700
Subject: [PATCH 771/909] Revert "arm64: dts: r8a77990: Tie Audio-DMAC to
 IPMMU-MP"

commit 80359ee925335de7209f23742bbf78aedf3d89fb from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This reverts commit eb7141f4463f52e72ebda5513ea8ce1229d3e23c
to keep IPMMU address translation for Audio-DMAC is disabled
by default.

Signed-off-by: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990.dtsi | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990.dtsi b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
index ebd7fbbde65e..295f6a1fa8c0 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
@@ -1458,14 +1458,6 @@
 			resets = <&cpg 502>;
 			#dma-cells = <1>;
 			dma-channels = <16>;
-			iommus = <&ipmmu_mp 0>, <&ipmmu_mp 1>,
-				 <&ipmmu_mp 2>, <&ipmmu_mp 3>,
-				 <&ipmmu_mp 4>, <&ipmmu_mp 5>,
-				 <&ipmmu_mp 6>, <&ipmmu_mp 7>,
-				 <&ipmmu_mp 8>, <&ipmmu_mp 9>,
-				 <&ipmmu_mp 10>, <&ipmmu_mp 11>,
-				 <&ipmmu_mp 12>, <&ipmmu_mp 13>,
-				 <&ipmmu_mp 14>, <&ipmmu_mp 15>;
 		};
 
 		adsp: adsp@ec800000 {
-- 
2.17.1

